### DISCUSSION TITLE
Problem in FAQ for flank definition

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#12] [Q&A] [answered]
I think the letter to left of S is I, not W in [FAQ](https://eagerwords.com/faq.html)

Figure 1. The flanks of point (row: 4, column: 5) containing 'S'.

up: [R] (opposing)
down: []
left: [W] (friendly)
right: [S, H] (friendly)

**Comment by azadbolour** [comment, marked as answer]
Thanks. The flanks are incorrectly reported in the FAQ. They should be:

   up: [R] (opposing)
    down: []
    left: [W, I] (friendly)
    right: [H] (friendly)

**Comment by azadbolour** [comment]
Should not be fixed on the beta test site. Slated for deployment May 2023.
